UnrealEngine.gitignore

```diff
@@ -40,6 +43,7 @@
 *.sdf
 *.VC.db
 *.VC.opendb
+CMakeLists.txt
```

Comment:
We have an existing [`CMake.gitignore`](https://github.com/github/gitignore/blob/master/CMake.gitignore) file that includes this rule, and I feel like duplicating the rule here isn't helpful in the long-term.